srcalc/src/main/java/gov/va/med/srcalc/domain/model/Procedure.java

```diff
@@ -86,6 +86,9 @@ public Procedure(
         setEligible(eligible);
     }
 
+    /**
+     * The unique id of this procedure (not the CPT code).
```

Comment:
Other `getId()` methods say "The object's surrogate primary key. Don't show this to the user.", which is more informative. ("Surrogate key" has a more precise meaning than "unique id".)